manager/integration/tests/test_rwx.py

```diff
@@ -30,7 +30,7 @@ def write_data_into_pod(pod_name_and_data_path):
                                  DATA_SIZE_IN_MB_3)
 
 
-def test_rwx_with_statefulset_multi_pods(core_api, statefulset):  # NOQA
+def test_rwx_with_statefulset_multi_pods(nfs, core_api, statefulset):  # NOQA
```

Comment:
If you're using the fixture, no need to call it for single test. Module here is all the tests in the file test_rwx, so the fixture with scope module will run automatically before starting of any test in this file.

Reply:
Hello @khushboo-rancher, I kept import fixture nfs and removed all nfs function call.